### PR TITLE
Works & about me section

### PR DESCRIPTION
**Pull request by VelzckC0D3**
# My Portfolio

# What I have submitted
- The `#Works` section trying to make it as similar as I could (as it's requested by Microverse)
- The `#AboutMe` section where you are able to see almost all the website finished.
- I optimized some CSS code and the font loading with HTML
- Netlify live demo updated! [Click here to access it!](https://velzckcode.netlify.app/)
# ❗♻️ New update | requested changes ♻️ ❗ 
- As my first reviewer noticed I was having semantic errors on the `See more` buttons on the grid, I corrected it for all 7 buttons!   
- Also, corrected the semantic error of having 2 `h1` on the same webpage to optimize the SEO! 

# ⬇️ New Features! ⬇️
✅ A very nailed replica of the mobile version of [Microverse Portfolio Template #2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=JqSXbNsYkocPTz9n-0)
✅ Added "Works" section where you can see that I implemented Grid on the layout in order to accomplish the requirements
works

✅ Added "About Me" section. All this design is really adaptative to almost all screen sizes, I'm excited to continue with it! 
aboutMe

✅ Included `:hover` animations for all the buttons on the design! 
hoverIcons

# ✨ Thank you for your review! ✨
ThumbsUpMetalGearGIF

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,7 +56,241 @@ <h2 class="roboto">I'm a Software Developer</h2>
...
+                                                href="https://developer.mozilla.org/en/docs/Web/HTML" target="_blank"
+                                                rel="noopener noreferrer">html</a></li>
+                                        <li class="cardLi"> <a class="cardLink" href="https://getbootstrap.com/"
+                                                target="_blank" rel="noopener noreferrer">bootstrap</a></li>
+                                        <li class="cardLi"> <a class="cardLink" href="https://www.ruby-lang.org/es/"
+                                                target="_blank" rel="noopener noreferrer">Ruby</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+                            <div class="gridButton">see project</div>
```
Comment (brainconnect93) [inline index.html]: - [x] Hi @VelzckC0D3, I must really commend your effort, in short, you have a very wonderful project, 👍 👍 I would have loved to approve this project at this your first submission but I really want to make this your project looks great semantically by requesting these changes.
  - Please I would suggest you make all the `See project` button wrapped inside of a `button` or an `a` tag for semantic reasons.
  E.g:
```HTML
<div class="gridButton"><a hef="#">see project</a></div> OR
<div class="gridButton"><button type="submit">see project</button></div>
```

File: index.html [hunk 2]
```diff
@@ -60,7 +56,241 @@ <h2 class="roboto">I'm a Software Developer</h2>
...
+                        <h2>About me</h2>
+                    </div>
+                    <div id="aboutText" class="inter">
+                        <p>Hello I’m a software developer! I can help you build a product , feature or website Look
+                            through some of my work and experience! If you like what you see and have a project you need
+                            coded, don’t hestiate to contact me.</p>
+                    </div>
+                    <div id="buttonCV">
+                        <a id="cvLink" class="inter" href="http://github.com/velzckC0D3" target="_blank" rel="noopener noreferrer">
+                            <h1>get my resume</h1>
```
Comment (brainconnect93) [inline index.html]: - [x] Please kindly note that for best practice we can only use one `h1` in a webpage and this is for SEO (search engine optimization) and accessibility reasons. I would kindly suggest you remove this `h1` tag and leave your `Get my resume` button to be wrapped inside of an `anchor` tag alone. 👏 👏

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @VelzckC0D3, 👋 

I must say this you have a fantastic work in your project, you have shown an high level of professionalism, and I will be sincere with you you are going places in this space as a software developer, KEEP IT UP!!! 🍾 🍾 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (VelzckC0D3) [reply index.html]: Hi, @brainconnect93, thank you so much for your words! I really, really appreciate that. I worked on it right away and solved it for all 7 buttons! 
image

File: index.html (hunk 2 above)
Comment (VelzckC0D3) [reply index.html]: Got it! I toke out the ``Get my resume`` out of the H1 and now it's only inside the `anchor` I give it the properties with CSS and it's done! 
image

File: index.html [hunk 3]
```diff
@@ -60,7 +56,248 @@ <h2 class="roboto">I'm a Software Developer</h2>
                     </ul>
                 </div>
             </div>
+        </section>
```
Comment (bobb-Rob) [inline index.html:59]: A very beautiful project you here. Well done 👏🏾 🍾 

I see a few things that can further take you higher. 

- #### First Project Button
You currently have a colored border on this button. Following the Figma template, thiere should be not border. Consider this. 

Your Work | Figma Design
--- | ---
image | image

- #### Commit message
Getting in the habit of creating quality commit messages makes using and collaborating with Git a lot easier. You will sure find very useful tips in [this article ](https://initialcommit.com/blog/git-commit-messages-best-practices) that will help you improve your commit messages.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @VelzckC0D3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (VelzckC0D3) [reply index.html:59]: Thank you!! I appreciate your feedback, right now I'm going to eat that article !!

**Pull request merged**